### ISSUE TITLE
Must-gather is failing with unable to pull image: ImagePullBackOff

### ISSUE THREAD

**Issue by am-agrawa** [red-hat-storage/ocs-ci#6842]
```
2022-12-14 22:55:18  error: gather did not start for pod must-gather-ptdxr: unable to pull image: ImagePullBackOff: Back-off pulling image "registry.redhat.io/odf4/ocs-must-gather-rhel8:latest"
2022-12-14 22:55:18  
2022-12-14 22:55:18  17:25:08 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-storage get pods -o name
2022-12-14 22:55:18  17:25:09 - MainThread - ocs_ci.ocs.utils - ERROR  - 
2022-12-14 22:55:18  17:25:09 - MainThread - ocs_ci.ocs.utils - ERROR  - Failed during must gather logs! Error: Error during execution of command: oc adm must-gather --image=registry.redhat.io/odf4/ocs-must-gather-rhel8:latest --dest-dir=/home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1670999226/test_add_capacity_with_resource_delete[workload_storageutilization_rbd0-osd-0-False]_ocs_logs/ocs_must_gather.
2022-12-14 22:55:18  Error is 
2022-12-14 22:55:18  
2022-12-14 22:55:18  Error running must-gather collection:
2022-12-14 22:55:18      gather did not start for pod must-gather-ptdxr: unable to pull image: ImagePullBackOff: Back-off pulling image "registry.redhat.io/odf4/ocs-must-gather-rhel8:latest"
2022-12-14 22:55:18  
2022-12-14 22:55:18  Falling back to `oc adm inspect clusteroperators.v1.config.openshift.io` to collect basic cluster information.
```

Failed for this and many other tests- https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/362/7556/331763/331845/331846/log?item1Params=filter.eq.hasStats%3Dtrue%26filter.eq.hasChildren%3Dfalse%26filter.in.type%3DSTEP%26filter.in.status%3DFAILED%252CINTERRUPTED

**Comment by am-agrawa** [comment]
@OdedViner  Do you know this issue?

**Comment by am-agrawa** [comment]
The failure was with ODF 4.12 on IBM cloud and is fixed by #6893. Closing this issue but could be re-opened if the issue is hit again when we have the IBM account created.
